package.json

```diff
@@ -11,7 +11,9 @@
   "types": "build/index.d.ts",
   "scripts": {
     "build": "run-s check-types build:css build:app",
-    "build:css": "cross-env NODE_ENV=production tailwindcss -i src/tailwind.css -o src/main.css",
+    "build:css": "run-s build:css:application build:css:renderer",
+    "build:css:application": "cross-env NODE_ENV=production tailwindcss -c application/tailwind.config.js -i application/input.css -o application/main.css",
+    "build:css:renderer": "cross-env NODE_ENV=production tailwindcss -c tailwind.config.js -i src/tailwind.css -o src/main.css",
```

Comment:
Tw noob question: Since this builds tw for production, would tw need to be configured to compile when running dev server too?

Reply:
yeah you are right, configured in https://github.com/TradeTrust/generic-templates/pull/92/commits/4b822c253d52c28fdd9d94e741f75615bf1c9fe2.

note there are 2 tracks of css, 1 in example application (low priority) and 1 in renderer itself (critical priority).

Reply:
Cool! Thanks!